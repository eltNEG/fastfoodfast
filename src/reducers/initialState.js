import { FOOD_FETCH_STATES } from '../constants';

const initialState = {
  foods: {
    foodFetchState: FOOD_FETCH_STATES.notFetced,
    fetchFoodLoading: false,
    foodItems: []
  }
};

export default initialState;
