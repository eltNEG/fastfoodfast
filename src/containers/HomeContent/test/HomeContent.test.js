import React from 'react';
import { Provider } from 'react-redux';
import thunk from 'redux-thunk';
import configureStore from 'redux-mock-store';
import { shallow, mount } from 'enzyme';
import HomeContentContainer from '..';
import initialState from '../../../reducers/initialState';

describe('HomeContentContainer', () => {
  const mockStore = configureStore([thunk]);
  const store = mockStore({ foodReducer: initialState.foods });
  const doFetchFoods = () => {};
  describe('HomeContentContainer Snapshot', () => {
    it('should match snapshot', () => {
      const wrapper = mount(
        <Provider store={store}>
          <HomeContentContainer doFetchFoods={doFetchFoods} />
        </Provider>
      );
      expect(wrapper).toMatchSnapshot();
    });
  });
});
