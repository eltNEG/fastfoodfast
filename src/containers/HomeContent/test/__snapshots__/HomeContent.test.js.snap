// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeContentContainer HomeContentContainer Snapshot should match snapshot 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <Connect(HomeContentContainer)
    doFetchFoods={[Function]}
  >
    <HomeContentContainer
      doFetchFoods={[Function]}
      foodItems={Array []}
    >
      <HomeContent
        foodItems={Array []}
      >
        <div
          className="home-content"
        >
          <CardGroup
            className="food-list"
            doubling={true}
            items={Array []}
            itemsPerRow={4}
            stackable={true}
          >
            <div
              className="ui doubling stackable four cards food-list"
            />
          </CardGroup>
        </div>
      </HomeContent>
    </HomeContentContainer>
  </Connect(HomeContentContainer)>
</Provider>
`;
